### PR TITLE
(CAT-1345) Update gem README.md

### PR DESCRIPTION
**Pull request by david22swan**
README is barebones and contains little information. Updating to include more detailed instructions and what it is and how to use it.

## Checklist
- [x] 🟢 Spec tests.
- [x] 🟢 Acceptance tests.
- [ ] Manually verified.

### CODE REVIEW

**Review by LukasAud — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,18 +1,55 @@
...
+
 ```ruby
 builder = Puppet::Modulebuilder::Builder.new('./puppetlabs-motd', './pkg', nil)
 builder.build
 ```
 
-## Development
+For conveniances sake the `puppet-modulebuilder` gem has been included within the `PDK` and as such can be called on to run against a module from within it using the build command as shown below:
+
+```ruby
```
Comment (ekohl) [inline README.md:33]: This isn't ruby

File: README.md [hunk 2]
```diff
@@ -1,18 +1,55 @@
...
+
+## Overview
+
 The `puppet-modulebuilder` gem contains the reference implementation for building Puppet modules from source.
 
+## Description
+
+The purpose of this tool is to take a given local module directory and compile it into a `.tar` file, known as the `tarball`, that can then be installed directly by Puppet on a target machine or uploaded onto the [Puppet Forge](https://forge.puppet.com/) so that it can be accessed publicly.
+
+As part of this process any non-deliverable aspects of the module, parts of it related to the modules development or testing for example, are stripped away leaving only the documentation and the puppet/ruby code that is needed for the module to function.
```
Comment (ekohl) [inline README.md:20]: Should it describe which files it reads here? Like `.gitignore`, `.pmtignore`, `.pdkignore` and in which order?

**Review by ekohl — COMMENTED** [review]
I like the expansion. Bit late, but please consider my suggestions for a follow up.

File: README.md (hunk 2 above)
Comment (david22swan) [reply README.md:20]: Created a new PR with an update: https://github.com/puppetlabs/puppet-modulebuilder/pull/62

**Pull request merged**